### DISCUSSION TITLE
Récupération quotidienne de la consomation

### DISCUSSION THREAD

**Discussion by jrmedc** [bokub/conso-api#10] [Aide]
Bonjour, 

J'ai une question concernant la Disponibilité des données: 
Dans [cette partie de la page d'accueil]([Disponibilité des données](https://conso.boris.sh/#disponibilit%C3%A9-des-donn%C3%A9es)) il est indiqué que les donnée des compteurs ne sont envoyés qu'une fois par jour par les compteur linky, 
Est-ce une limitation structurelle de l'api d'enedis ?Une limitation lié à l'utilisation de cette API conso.boris? Ou alors  quelque chose que le client d'enedis peut paramétrer si il en a envie ? 
Je n'ai rien trouvé à ce sujet dans la documentation des APIs d'enedis et je me posais la question 

Merci par avance!
Bonne soirée

**Comment by bokub** [comment]
Salut ! 

Quand je dis "les donnée des compteurs ne sont envoyés qu'une fois par jour par les compteur linky", ça n'est pas une métaphore ! C'est n'est pas qu'une simple limitation d'API, mais une limitation qui vient de la conception des compteurs Linky et de toute l'infrastructure qui les supporte, et non ça n'est pas configurable

J'imagine que ça permet à Enedis de limiter la volumétrie des données qu'ils ont à traiter, une requête par jour c'est plus simple pour eux que de traiter du temps réel... :shrug:

**Comment by jrmedc** [comment]
Okay top merci beaucoup pour la réponse  !
